src/main/java/info/blockchain/wallet/contacts/data/Contact.java

```diff
@@ -113,4 +148,19 @@ public Contact fromQueryParameters(Map<String, String> queryParams){
 
         return contact;
     }
+
+    public String createURI() throws URISyntaxException {
+
+        List<NameValuePair> qparams = toQueryParameters();
+
+        qparams.add(new BasicNameValuePair("id", outgoingInvitation.getId()));
+
+        URIBuilder builder = new URIBuilder()
+                .setScheme("blockchain")
```

Comment:
https://github.com/blockchain/My-Wallet-V3-Android/blob/feature/metadata/app/src/main/java/piuk/blockchain/android/data/metadata/MetaDataUri.java may be of use